modular_skyrat/modules/reagent_forging/code/forge_items.dm

```diff
@@ -41,14 +41,17 @@
 /obj/item/forging/incomplete
 	name = "parent dev item"
 	desc = "An incomplete forge item, continue to work hard to be rewarded for your efforts."
```

Comment:
per discord, this should probably mention that you need a workbench to finish it